### PR TITLE
Update example image with different colours for different sections

### PR DESCRIPTION
**Pull request by vedhav**
Updates the example screenshot

### CODE REVIEW

File: vignettes/teal.Rmd [hunk 1]
```diff
@@ -47,11 +47,11 @@ if (interactive()) {
 
 As shown in the image above, this application consists of several distinct areas:
 
-* Application header: the title of the application shown at the top.
-* `teal` modules (bar at the top): in this case a simple module named "example teal module".
-* Encoding panel (panel on the left hand side): Module specific UI components, in this case a drop-down to select a dataset name.
-* Main output panel (panel on the middle): The outputs of the module, for this example module the chosen dataset is displayed.
-* Filter panel (panels on the right hand side): for filtering the data to be passed into all `teal` modules.
+* <span style="color: #FB6251;">Application Header</span> (title at the top): is the title of the application.
+* <span style="color: #3A88FE;">Teal Modules</span> (bar at the top): in this case a simple module named "example teal module".
```
Comment (chlebowa) [inline vignettes/teal.Rmd:51]: There is "bar at the top" (below) and now also "title at the top". This is confusing.

Change "title" to "banner"? Or leave "title at the top" and change "bar at the top" to "bar under the title"?

**Review by chlebowa — CHANGES_REQUESTED** [review]
Looks really great!

File: vignettes/teal.Rmd (hunk 1 above)
Comment (vedhav) [reply vignettes/teal.Rmd:51]:
```suggestion
* <span style="color: #FB6251;">Application Header</span> (title at the top): is the title of the application.
* <span style="color: #3A88FE;">Teal Modules</span> (bar under the title): in this case a simple module named "example teal module".
```

**Review by chlebowa — APPROVED** [review]
Good addition :+1:

**Pull request merged**